src/lib.rs

```diff
@@ -88,21 +90,39 @@ fn available_languages(config: &fpm::Config) -> fpm::Result<String> {
 }
 
 fn original_package_status(config: &fpm::Config) -> fpm::Result<String> {
-    let path = config.root.join("FPM/translation/original-status.ftd");
-    Ok(if path.is_file() {
-        std::fs::read_to_string(path)?
-    } else {
-        include_str!("../ftd/translation/original-status.ftd").to_string()
-    })
```

Comment:
The rest of the function should move to the else clause. If the file is not found then we auto generate the file.